### ISSUE TITLE
project.html images are stretched

### ISSUE THREAD

**Issue by ryanfortner** [Crilum/crilum.github.io#1]
![image](https://user-images.githubusercontent.com/71036629/171686944-fadc38b7-3666-43a3-8758-289886870422.png)

btw, I like your website, its' really unique!

**Comment by Crilum** [comment]
Yeah, I know.... I've tried to fix it a few times, but it keeps breaking... lol. I'll try to fix it today.

**Comment by Crilum** [comment]
So, for some reason, it works in Firefox, but in Chromium, it's stretched... I assume you're using Chrome(ium)?

Firefox:

![Screenshot from 2022-07-01 14-05-03](https://user-images.githubusercontent.com/91354257/176961968-8075a27c-1609-490f-85c2-d34ecd9ab476.png)

Chromium:
![Screenshot from 2022-07-01 14-06-11](https://user-images.githubusercontent.com/91354257/176961993-3b5e86da-4669-4863-8589-fa8a0683c66e.png)
